dev/Microsoft.Process.Environment/EnvironmentManager.cpp

```diff
@@ -63,4 +74,125 @@ namespace winrt::Microsoft::ProjectReunion::implementation
     {
         throw hresult_not_implemented();
     }
+
+    StringMap EnvironmentManager::GetProcessEnvironmentVariables()
+    {
+        //Get the pointer to the process block
+        LPWSTR environmentVariablesString = GetEnvironmentStrings();
+        THROW_HR_IF_NULL(E_POINTER, environmentVariablesString);
+
+        // parse the block into a map
+        LPTSTR variable;
+        variable = (LPTSTR)environmentVariablesString;
```

Comment:
Why the cast?

1. Use C++ new-style cast notation (const_cast etc). Don't use C style casts.
2. Cast shouldn't be necessary